CadRevealFbxProvider/FbxImporter.cs

```diff
@@ -15,11 +15,6 @@ public FbxNode LoadFile(string filename)
         {
             throw new FileNotFoundException(filename + " was not found");
         }
-        //this should never be called in the first place if the SDK is invalid
-        if (!_sdk.IsValid())
-        {
-            throw new NullReferenceException("Cannot load files with invalid SDK.");
-        }
```

Comment:
Why is this check removed?

Reply:
There is another call to this function in FbxWorkload (ReadFbxData), where it checks this before calling the LoadFile function of the FbxImporter